### PR TITLE
Add errors check

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

File: context/copy.go [hunk 1]
```diff
@@ -93,7 +93,9 @@ func (ctx *Context) CopyPackage(destPath, srcPath, lookRoot, pkgPath string, ign
 	}
 	if h != nil {
 		// Write relative path to GOPATH.
-		h.Write([]byte(strings.Trim(pkgPath, "/")))
+		if _, err := h.Write([]byte(strings.Trim(pkgPath, "/"))); err != nil {
```
Comment (kaey) [inline context/copy.go]: this looks unnecessary https://godoc.org/hash#Hash
```
// Write (via the embedded io.Writer interface) adds more data to the running hash.
// It never returns an error.
```

File: context/license.go [hunk 2]
```diff
@@ -177,7 +177,9 @@ func licenseCopy(root, startIn, vendorRoot, pkgPath string) error {
 			return errors.Errorf("Source license path doesn't exist %q", srcPath)
 		}
 		destDir, _ := filepath.Split(destPath)
-		os.MkdirAll(destDir, 0777)
+		if err = os.MkdirAll(destDir, 0777); err != nil {
+			return err
```
Comment (kaey) [inline context/license.go]: errors.Wrapf probably

File: context/copy.go (hunk 1 above)
Comment (mgrachev) [reply context/copy.go]: Fixed.

File: context/license.go (hunk 2 above)
Comment (mgrachev) [reply context/license.go]: Fixed.

**Comment by mgrachev** [comment]
@kardianos Do you have any comments?

**Pull request merged**